app/assets/stylesheets/application.scss

```diff
@@ -56,9 +63,55 @@ h6 {
   h3 {color: black;
     text-transform: uppercase;
     font-style: normal;
+    a {
+      color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.